app/dashboard/tasks/[filter]/TaskFilteredIndex.tsx

```
@@ -0,0 +1,98 @@
+      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
+        {data &&
+          data.map((task, index) => (
+            <TaskCard
+              id={task.id}
+              key={task.id}
+              name={task.name}
+              description={task.description}
+              completed={false}
+              tags={task.tags ?? []}
+            />
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)